rollup-common.js

```diff
@@ -39,7 +39,23 @@ const skipBundleOutput = {
   },
 };
 
-export function litRollupConfig({ entryPoints, external = [] } = options) {
+// Any private properties which we share between different _packages_ are
+// hard-coded here because they must never change between versions. Mangled
+// names are randomly chosen uppercase letters, in case we ever might want to
+// use lowercase letters for short, public APIs.
+const crossPackagePropertyMangles = {
+  _createElement: "Y",
```

Comment:
We should consider prefixing these, like with `$`

Reply:
It's not a slam-dunk idea, but it'd be to clearly mark that a property is special in case someone's looking at these objects in devtools.

Reply:
They aren't _that_ special though. They're just APIs that we happened to need to share across the internal package boundaries we invented?

Reply:
They're special compared to public properties. We don't want other code to use these properties.

Reply:
Right, but they're not too special compared to other minified properties, which is what a `$` would distinguish them from?

Reply:
Going to merge to move forward, but we can revisit this in a follow up if needed.